pom.xml

```diff
@@ -60,25 +60,25 @@
     </organization>
 
     <properties>
-        <assertj-core.version>3.22.0</assertj-core.version>
-        <jackson.version>2.13.2.1</jackson.version>
+        <assertj-core.version>3.24.2</assertj-core.version>
+        <jackson.version>2.14.0</jackson.version>
```

Comment:
Should we move to 2.14.2 directly @jhaeyaert @guillaumelamirand ? For now, I used the version defined in the vertx-bom

Reply:
I would say go with 2.14.2, most of the time patch version are security fixes. I let Jeoffrey gives his opinion.

Reply:
Agree